bootloader/Makefile

```diff
@@ -0,0 +1,12 @@
+APPLICATION = bootloader
```

Comment:
Well, I also don't think that adding it to examples is the good place... For me it looks good here because it states an essential part of RIOT, though it's optional...